app/services/proofing/lexis_nexis/instant_verify/proofer.rb

```diff
@@ -49,7 +49,7 @@ def parse_verification_errors(verification_response)
         # rubocop:disable Layout/LineLength
         def failed_result_can_pass_with_additional_verification?(verification_response)
           return false unless verification_response.verification_status == 'failed'
-          return false unless verification_response.verification_errors.keys.sort == [:"Execute Instant Verify", :base]
+          return false unless verification_response.verification_errors.keys.sort == [:InstantVerify, :base]
```

Comment:
this is the most counterintuitive sorting I've ever seen! but confirmed it's accurate. Another idea would be to `.to_set` both sides order does't matter
```suggestion
          return false unless verification_response.verification_errors.keys.to_set == Set[:InstantVerify, :base]
```